portfolio/src/main/webapp/script.js

```diff
@@ -158,12 +158,33 @@ async function checkLogin(){
 
 /** Creates a map and adds it to the page. */
 function initMap() {
-  const nightMapStyle = new google.maps.StyledMapType(nightVersion, {name: 'Night'});
+  const jakarta = {lat: -6.175540, lng: 106.82743};
+
   const map = new google.maps.Map( document.getElementById("map"), {
-    center: {lat: -6.175540, lng: 106.8274392}, 
+    center: jakarta,
     zoom: 15,
     mapTypeControlOptions: { mapTypeIds: ['roadmap', 'satellite', 'hybrid', 'terrain', 'night_map'] }
   });
+
+  /** Set up map in night mode */
```

Comment:
Could you please change the comments to `/* … */`? JSDoc (`/** … */`) style comment shouldn't be used for implementation comments ([ref](https://google.github.io/styleguide/jsguide.html#formatting-block-comment-style)).